rfd/0008-application-access.md

```diff
@@ -0,0 +1,109 @@
+---
+authors: Russell Jones (rjones@gravitational.com)
+state: draft
+---
+
+# RFD 8 - Teleport Application Access
```

Comment:
I would maybe change this to an RFD about Teleport Application Access JWTs vs a RFD for the feature/product in general

Reply:
I am slowly adding all of that in. For example, I added configuration information in today.